.devcontainer/devcontainer.json

```
@@ -27,7 +27,7 @@
     "esbenp.prettier-vscode",
     "xaver.clang-format"
   ],
-  "postCreateCommand": "cd /build/bun; bash /build/getting-started.sh; code /build/README.md",
+  "postCreateCommand": "cd /build/bun; bash /build/getting-started.sh; cat /build/README.md",
```

Sometimes the `code` executable will be in $PATH 

How about we try it if it works and fallback to cat:
```suggestion
  "postCreateCommand": "$(which code || which cat) /build/bun; bash /build/getting-started.sh; cat /build/README.md",
```

---

`code` should indeed be available inside the container at `/usr/local/bin/code`, however for some reason it was still causing an error to have it in the `postCreateCommand`, both when running from the CLI and nested within a VSCode instance. It might just be a problem with my system, though...

---

if it didn't work then that's that and we should `cat` it